tests/client/client.rs

```diff
@@ -0,0 +1,21 @@
+use bson::Bson;
+use mongodb::client::MongoClient;
+
+#[test]
+fn list_databases() {
```

Comment:
That behavior is very much version dependent, you may be looking at an older version of the python driver or using mongodb <= 2.6. For instance, [the node driver explicitly checks if the server has a list collections command, and returns a cursor if it does](https://github.com/mongodb/node-mongodb-native/blob/47dbb499283ba9887bc7602a743d4719f6332b46/lib/db.js#L477-488).